Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM luxonis/depthai-ros-rae AS builder
 ARG SIM=0
 ARG CORE_NUM=1
 ARG BUILD_TYPE="RelWithDebInfo"
+ARG INCLUDE_SPECTACULARAI_ROS=YES
```

Comment:
I would set that to false and modify the workflow file to set that argument to true